docs/notebooks/getting_started.ipynb

```diff
@@ -0,0 +1,1918 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    adata.layers["counts_per_million"] = sc.pp.normalize_total(adata, target_sum=10**6, layer="raw", inplace=False)["X"]
</pre><p> I would maybe change to simplify a little:</p><pre class="ql-syntax" spellcheck="false">adata.layers['counts_per_million'] = adata.layers['raw'].copy()
sc.pp.normalize_total(adata, target_sum=10**6, layer="raw")
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/scverse/scverse-tutorials/pull/34/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='26'/>

Reply:
<p>good point, changed and added an extra line to explain the layer=.. argument</p>